src/geojsonvt.cpp

```diff
@@ -44,17 +44,14 @@ GeoJSONVT::convertFeatures(const std::string& data, uint8_t maxZoom, double tole
 
 const Tile GeoJSONVT::emptyTile {};
 
-GeoJSONVT::GeoJSONVT(std::vector<ProjectedFeature> features_,
-                     uint8_t maxZoom_,
-                     uint8_t indexMaxZoom_,
-                     uint32_t indexMaxPoints_,
-                     bool solidChildren_,
-                     double tolerance_)
-    : maxZoom(maxZoom_),
-      indexMaxZoom(indexMaxZoom_),
-      indexMaxPoints(indexMaxPoints_),
-      solidChildren(solidChildren_),
-      tolerance(tolerance_) {
+GeoJSONVT::GeoJSONVT(std::vector<ProjectedFeature> features_, Options options)
```

Comment:
Why not store an `Options` object directly into the `GeoJSONVT` object rather than exploding it into its parts?